packages/openapi-to-graphql/test/authentication.test.ts

```diff
@@ -147,12 +147,13 @@ test('Get project using API key passed as option - viewer is disabled', async ()
 })
 
 test('Get project using API key passed in the requestOptions - viewer is disabled', async () => {
-  let { schema } = await openapiToGraphql.createGraphQlSchema(oas, {
+  const { schema } = await openapiToGraphql.createGraphQlSchema(oas, {
     viewer: false,
     requestOptions: {
       headers: {
         access_token: 'abcdef'
-      }
+      },
+      url: undefined
```

Comment:
This is kind of ugly... not sure if there's much we can do about it. We rely on the request module for this object type definition.

Reply:
That's an ok price to pay, IMO.